gnovm/pkg/gnolang/store.go

```diff
@@ -6,12 +6,17 @@ import (
 	"strconv"
 	"strings"
 
+	lru "github.com/hashicorp/golang-lru/v2"
+
 	"github.com/gnolang/gno/tm2/pkg/amino"
 	"github.com/gnolang/gno/tm2/pkg/std"
 	"github.com/gnolang/gno/tm2/pkg/store"
 )
 
 const iavlCacheSize = 1024 * 1024 // TODO increase and parameterize.
+const objectCacheSize = 100       // TODO parameterize.
+const typeCacheSize = 1000        // TODO parameterize.
+const nodeCacheSize = 10000       // TODO parameterize.
```

Comment:
How did you evaluate those values ?

Reply:
Executing a stress test using supernova. Ideally, we should define a total max memory available for store cache.